src/include/uapi/ipc.h

```diff
@@ -40,6 +40,7 @@
 #define __INCLUDE_UAPI_IPC_H__
 
 #include <uapi/abi.h>
+#include <sof/io.h>
```

Comment:
why is this needed? To get the BIT() definition?

Reply:
Yes, it is for BIT() macro.